.nvmrc

```diff
@@ -1 +1 @@
-22.11.0
+v22.17.0
```

Comment:
did you mean to add in the "v"? Can we fix the format to terminate with a newline?

Reply:
No worries - don't mind, the "v" was new and wasn't sure if it got parsed correctly